plugin/src/main/java/org/wildfly/plugin/provision/AbstractProvisionServerMojo.java

```diff
@@ -157,9 +157,9 @@
     private File provisioningFile;
 
     /**
-     * The target directory the application to be deployed is located.
+     * The target directory. Read-only.
      */
-    @Parameter(defaultValue = "${project.build.directory}/", property = PropertyNames.DEPLOYMENT_TARGET_DIR)
+    @Parameter(defaultValue = "${project.build.directory}/", readonly = true)
     protected File targetDir;
```

Comment:
Not really a big deal, but you could just remove this and then below using `project.getBuild().getDirectory()`.

Reply:
@jamezp , thank-you. Fixed.